apps/emqx_management/test/emqx_mgmt_clients_api_SUITE.erl

```diff
@@ -62,6 +62,29 @@ t_clients(_) ->
     ?assertEqual(ClientsLimit, emqx_mgmt:max_row_limit()),
     ?assertEqual(ClientsCount, 2),
 
+    timer:sleep(300),
+
+    %% get /clients with time(rfc3339)
+    NowTimeStampInt = erlang:system_time(millisecond),
+    %% Do not uri_encode `=` to `%3D`
+    NowTimeString   = emqx_http_lib:uri_encode(binary:bin_to_list(emqx_mgmt_api_clients:unix_ts_to_rfc3339_bin(NowTimeStampInt))),
+    Parameters      = [Param ++ NowTimeString
+                       || Param <- [ "lte_created_at="
+                                   , "lte_connected_at="
+                                   , "gte_created_at="
+                                   , "gte_connected_at="]],
+    RequestResults  = [emqx_mgmt_api_test_util:request_api(get, ClientsPath, Param, AuthHeader)
+                       || Param <- Parameters],
+    DecodedResults  = [emqx_json:decode(Response, [return_maps])
+                       || {ok, Response} <- RequestResults],
+    [ok, ok] = [?assert(
```

Comment:
It doesn't make sense to check the result of `?assert` macro, it throws an exception when assert fails.

Reply:
Thanks. It's not necessary.
The language server was warning unused list. So I changed code but forgot remove them.
The result check would be remove in later commit.

Reply:
In this case you can use `lists:foreach()` function instead of list comprehension, it doesn't return the result.